proto/rpc/v1/xrp_ledger.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package rpc.v1;
+
+import "rpc/v1/account_info.proto";
+import "rpc/v1/fee.proto";
+import "rpc/v1/submit.proto";
+import "rpc/v1/tx.proto";
+
+
+// RPCs available to interact with the XRP Ledger.
+service XRPLedgerAPIService {
```

Comment:
No error handling so far, I guess that'll come later?

Reply:
Could you elaborate on what you'd expect to see here? 

Obviously, gRPC supports plain error codes and descriptions. There's the advanced error model, which isn't really standard (yet): https://cloud.google.com/apis/design/errors#error_model. 

If we really needed to encode error details, we could always roll them into the response metadata?

Or are you asking for their to be a generic `ResponseMessage`, that looks like:

```
message Response {
   oneof  {
      google.protobuf.Any response = 1
      google.rpc.Status error = 2;
   }
}

message RippledError {
  // ...
}
```

Reply:
Error handling is built into grpc in a way. I'm using these status codes, along with string messages: [https://github.com/grpc/grpc/blob/master/doc/statuscodes.md](https://github.com/grpc/grpc/blob/master/doc/statuscodes.md)
If you look at the c++ handler code, you can see how I'm handling errors. For instance, if one passes a non-existent account to the GetAccountInfo RPC, I return status code NOT_FOUND, with message "account not found". gRPC will return this status code and message instead of the defined response type.

Reply:
@MarkusTeufelberger lmk what you think here or just resolve this

Reply:
This is a comment, it can't be "resolved" on GitHub. Also I'm not in the Ripple team, so I also don't have some moderation tools available that project owners might have.

If the response to errors looks like in JSONRPC I guess it's fine.